src/jit/lsra.cpp

```diff
@@ -3596,7 +3602,14 @@ void LinearScan::buildRefPositionsForNode(GenTree*                  tree,
     int consume = info.srcCount;
     int produce = info.dstCount;
 
+#ifndef ARM_SOFTFP
     assert(((consume == 0) && (produce == 0)) || (ComputeAvailableSrcCount(tree) == consume));
+#else
+    // TODO-ARM-Cleanup: we need to change assert condition.
+    // For example, count and dst count of `vmov.d2i` do not match.
+    // The below call is temporally added just to supress warning message.
+    ComputeAvailableSrcCount(tree);
```

Comment:
Please take a look at [armel CI failure](https://ci.dot.net/job/dotnet_coreclr/job/master/job/armel_cross_release_tizen_prtest/3570/console).
You can use it with `DEBUG` defined or in `assert`, because `ComputeAvailableSrcCount()` is defined only when `DEBUG` is defined.